src/index.ejs

```diff
@@ -48,12 +48,6 @@
         if (options.async === "false") {
           getToken(createSDK)
         }
-        else if (options.token) {
-          createSDK(options.token)
-        }
-        else if (options.mobileFlow) {
-          window.onfidoOut = Onfido.init({mobileFlow: true})
```

Comment:
So where do you set `mobileFlow` to `true` now?

Reply:
This is set in the hosted_web_sdk repo: https://bitbucket.org/onfido/hosted_web_sdk/src/4547923844801f855109d14892f0a73ac1efb334/production.html?at=master&fileviewer=file-view-default

This index.ejs is only used for development. The code I removed here is legacy from the proof of concept

Reply:
Oh yes, forgot about that.